pkg/kubectl/cmd/create_service.go

```diff
@@ -220,3 +221,59 @@ func CreateServiceLoadBalancer(f cmdutil.Factory, cmdOut io.Writer, cmd *cobra.C
 		OutputFormat:        cmdutil.GetFlagString(cmd, "output"),
 	})
 }
+
+var (
+	serviceExternalNameLong = templates.LongDesc(`
+	Create an ExternalName service with the specified name.`)
```

Comment:
Would it make sense to mention here what ExternalName services can be used for? Like "can be used for CNAME", etc. This may not be very clear for used in a first sight.